internal/model/role_course.go

```diff
@@ -48,12 +57,14 @@ func GetCourseUserRoleString(role CourseUserRole) string {
 	return courseRoleToString[role]
 }
 
-func GetAllCourseUserRoles() map[CourseUserRole]string {
-	return courseRoleToString
-}
+func GetCommonCourseUserRolesCopy() map[string]CourseUserRole {
+	commonCourseUserRoleCopy := make(map[string]CourseUserRole, len(commonStringToCourseUserRole))
+
```

Comment:
Nit.

I would remove this newline, so the visual block is just "make the copy".